qa/tasks/vstart_runner.py

```diff
@@ -1201,11 +1201,22 @@ def __del__(self):
         if test_path:
             shutil.rmtree(test_path)
 
+
+class LogRotate():
+    def __init__(self):
+        self.conf_file_path = os.path.join(os.getcwd(), 'logrotate.conf')
+        self.state_file_path = os.path.join(os.getcwd(), 'logrotate.state')
+
+    def run_logrotate(self):
+        remote.run(args=['logrotate', self.conf_file_path, '-s',
+                         self.state_file_path, '--verbose'])
+
 def teardown_cluster():
     log.info('\ntearing down the cluster...')
     remote.run(args=[os.path.join(SRC_PREFIX, "stop.sh")], timeout=60)
     remote.run(args=['rm', '-rf', './dev', './out'])
 
+
```

Comment:
whitespace churn

Reply:
Got used to adding 2 spaces before functions and classes at module level. Removed.